config/jail.conf

```diff
@@ -251,6 +251,12 @@ action_cf_mwl = cloudflare[cfuser="%(cfemail)s", cftoken="%(cfapikey)s"]
 #
 action_blocklist_de  = blocklist_de[email="%(sender)s", service="%(__name__)s", apikey="%(blocklist_de_apikey)s", agent="%(fail2ban_agent)s"]
 
+# Report ban via ipthreat.net.
+#
+# See action.d/ipthreat.conf for usage example and details
+#
+action_ipthreat = ipthreat
```

Comment:
This is not really needed... I know we have several entries with similar declaration, but:
- most of them have additional parameters which are either mandatory or pre-defined to some default value supplied from jails;
- and a comment describing where the action can be found is really enough (even that is unneeded in my opinion, many action doesn't noticed in jail.conf at all).

Also the definition is not more complex as without (even much shorter without that substitution):
```diff
 action = %(action_)s
-         %(action_ipthreat)s[ipthreat_flags="8",ipthreat_system="SSH", ipthreat_apikey=...]
+         ipthreat[ipthreat_flags="8",ipthreat_system="SSH", ipthreat_apikey=...]
```

Reply:
I've replaced the example in `config/action.d/ipthreat`, does this need to go in jail.conf instead?

Reply:
Although, I'm not against the comments, but don't see the necessity for `action_ipthreat = ipthreat`... I'd indeed prefer to remove everything. A look into `action.d/ipthreat.conf` is enough in my opinion.

> It could still be useful if someone is scrolling through that file or doing a search...

We can put a small notice about every action we have, but... let's avoid duplicating of everything.